src/mame/drivers/evmbug.cpp

```diff
@@ -140,13 +169,41 @@ void evmbug_state::evmbug(machine_config &config)
 	//TMS9902(config, "uart1", XTAL(12'000'000) / 4);
 }
 
+void tms9995bb_state::tms9995bb(machine_config &config)
+{
+	evmbug(config);
+	m_maincpu->set_addrmap(AS_PROGRAM, &tms9995bb_state::mem_map);
+}
+
 /* ROM definition */
 ROM_START( evmbug )
+	ROM_REGION( 0x1800, "maincpu", 0 )
+	ROM_LOAD( "u8.bin", 0x0000, 0x1000, CRC(ca869a70) SHA1(424d8d61ef15645e3ce3867c64a0cfb69633b5bc) )
+	ROM_LOAD( "u9.bin", 0x1000, 0x0800, CRC(7f71c9bf) SHA1(5215892585e5282650209c5ce13a2e4bd6041675) )
+ROM_END
+
+/* ROMs from http://www.stuartconner.me.uk/tms9995_breadboard/tms9995_breadboard.htm#eprom_binary_images
+and FF padded to make 32K */
+ROM_START( tms9995bb )
 	ROM_REGION( 0x8000, "maincpu", 0 )
-	ROM_LOAD( "evmbug.bin", 0x0000, 0x8000, CRC(a239ec56) SHA1(65b500d7d0f897ce0c320cf3ec32ff4042774599) )
+	ROM_SYSTEM_BIOS( 0, "evmbug", "EVMBUG system monitor")
+	ROMX_LOAD( "evmbug.bin",   0x0000, 0x8000, CRC(a239ec56) SHA1(65b500d7d0f897ce0c320cf3ec32ff4042774599), ROM_BIOS(0) )
+	ROM_SYSTEM_BIOS( 1, "basicram", "EVMBUG system monitor and BASIC in RAM")
+	ROMX_LOAD( "basicram.bin", 0x0000, 0x8000, CRC(6ed5aba3) SHA1(76e0e39c0c0028efca339fb3cebaf42351fadb94), ROM_BIOS(1) )
+	ROM_SYSTEM_BIOS( 2, "basicrom", "EVMBUG system monitor and BASIC in ROM")
+	ROMX_LOAD( "basicram.bin", 0x0000, 0x8000, CRC(ded6350b) SHA1(83cf64834e59e216a91065c01ec91be1e10e7244), ROM_BIOS(2) )
+    ROM_SYSTEM_BIOS( 3, "forth", "EVMBUG system monitor and Forth")
+	ROMX_LOAD( "forth.bin",    0x0000, 0x8000, CRC(eee8f390) SHA1(59fc2a23c9ac52dce09a519c784db378782242b1), ROM_BIOS(3) )
+    ROM_SYSTEM_BIOS( 4, "test1", "Test EPROM 1")
+	ROMX_LOAD( "test1.bin",    0x0000, 0x8000, CRC(9b110ffb) SHA1(58ee990fb17822a879442b98f1b78ccf86b79f00), ROM_BIOS(4) )
+	ROM_SYSTEM_BIOS( 5, "test2", "Test EPROM 2")
+	ROMX_LOAD( "test2.bin",    0x0000, 0x8000, CRC(e7a7832d) SHA1(aa8c29097033804d1a0abf2af7cd846edfbd71a3), ROM_BIOS(5) )
+	ROM_SYSTEM_BIOS( 6, "test3", "Test EPROM 3")
+	ROMX_LOAD( "test3.bin",    0x0000, 0x8000, CRC(a28579eb) SHA1(477f853970f132592714bcdd048ec932e96c8593), ROM_BIOS(6) )
```

Comment:
You’ve got mixed space and tab indentation here.

Reply:
fixed by 37f8602